src/command_modules/azure-cli-sf/setup.py

```diff
@@ -34,7 +34,7 @@
 DEPENDENCIES = [
     'azure-servicefabric==5.6.130',
     'azure-cli-core',
-    'adal==0.4.3'
+    'adal>=0.4.3'
```

Comment:
Correct, this was breaking dependency analysis in CI as there was already existing version 0.4.5. Hence we weakened the restriction to anything >= 0.4.3